graderx/views.py

```diff
@@ -153,11 +153,14 @@ def start_grading():
         lab_name = request.args['lab']
     except KeyError:
         return jsonify({'status': "course and lab query parameters must be included"}), 400
-    try:
+    # try:
+    if 'student' in request.args:
+        manager.run_grader(course_name, lab_name, student=True)
+    else:
         manager.run_grader(course_name, lab_name)
-        return "SUCCESS", 200
-    except:
-        return "Failed to run the grader", 500
+    return "SUCCESS", 200
```

Comment:
It's not advisable to return just a string from a JSON API since that's not a valid json object, you can instead return
```JSON
{
 "message": "success"
}
```

this is if you must return a message, otherwise you can just return `200 OK` without any content